### PR TITLE
feat: add lackluster colors tmux extra

### PR DESCRIPTION
**Pull request by lanjoni**
## Description

This tmux configuration file is inspired by [zenburn tmux extra](https://github.com/mcchrish/zenbones.nvim/tree/main/extras/tmux), with a simplier configuration.

To use it, you can just copy and paste inside your `~/.tmux.conf`.

### Screenshots
Screenshot 2024-06-13 at 18 26 10

### CODE REVIEW

**Comment by slugbyte** [comment]
Thanks so much for the contribution!

I think I might rename it to lackluster-snow.tmux because it has a light background, and eventually I want to make a `lackluster-snow` variant that is a light background version of the colorscheme.

**Comment by slugbyte** [comment]
I also found that is missing quite a few styles like `menu-style` 
Screenshot 2024-06-14 at 14 54 40

**Comment by slugbyte** [comment]
this is my custom non-lackluster theme that I'm using a reference to see what things are not set



```tmux
# THEME
# - black 0
# - white 16
# - pink 197
# - purple 105
# - dark gray 234
# - medium gray 240
# - light gray 245

set-option -g status-fg                     "colour0"
set-option -g status-bg                     "colour234"
set-option -g clock-mode-colour             "colour197"
set-option -g copy-mode-current-match-style "bg=colour105,fg=colour16"
set-option -g copy-mode-match-style         "bg=colour197,fg=colour16"
set-option -g menu-selected-style           'bg=colour105,fg=colour16'
set-option -g menu-style                    'bg=colour16,fg=colour197'
set-option -g message-command-style         'bg=colour197,fg=colour16'
set-option -g message-style                 'bg=colour16,fg=colour197'
set-option -g mode-style                    'bg=colour105,fg=colour16'
set-option -g pane-active-border-style      "fg=colour245"
set-option -g pane-border-lines             "heavy"
set-option -g pane-border-style             "fg=colour197"
set-option -g pane-border-style             "fg=colour234"
set-option -g popup-border-lines            "rounded"
set-option -g status-right-style            "fg=colour240"
set-option -g window-status-current-style   'fg=colour245'
set-option -g window-status-style           'fg=colour240'

# BAHAHAHA, I can't belive I was bored enough to figure out this bonkers FORMAT system!
# window stats is index or if has name index:name
set-option -g window-status-format         '#{?#{==:#W,},#I,#I:#W}'
set-option -g window-status-current-format '#{?#{==:#W,},#I,#I:#W}'
# status right will show the time in 24 hour format and if there is more than one
# server session the current session name will be there too
set-option -g status-right                 ' #{?#{>:#{server_sessions},1},(#{session_name}),} #(date +"%H:%M") '
set-option -g status-left                  ' '
```

File: extra/tmux/lackluster.tmux [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# This file is inspired by [zenbones tmux extra](https://github.com/mcchrish/zenbones.nvim/tree/main/extras/tmux)
```
Comment (slugbyte) [inline extra/tmux/lackluster.tmux:1]: can you add `copy-mode-current-match-style` `copy-mode-match-style` `menu-selected-style` `menu-style`, and `message-command-style`  and rename the file to `lackluster-snow.tmux`

**Review by slugbyte — CHANGES_REQUESTED** [review]
When I tested it I found a few bugs and missing styles, I left some notes on the code for you to fix.

If you patch these I'll be sure to merge the pr ASAP :)

Thanks so much for the help!

File: extra/tmux/lackluster.tmux [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+set -g status-left ' #[fg=#080808,bold]#{s/root//:client_key_table} '
+set -g status-right '#[fg=#080808,bold] [#S]#[fg=#080808,bold] [%d/%m] #[fg=#080808,bold][%I:%M%p] '
+set -g status-style fg='#080808',bg='#DEEEED'
```
[the remark this answers is not in the document]
Comment (lanjoni) [reply extra/tmux/lackluster.tmux]: thanks for testing this, i updated to lower case now

File: extra/tmux/lackluster.tmux (hunk 1 above)
Comment (lanjoni) [reply extra/tmux/lackluster.tmux]: shure! that's sounds perfect!

i'm using your configuration example to build `lackluster-snow` and `lackluster` tmux colors. what do you think about it?

**Review by slugbyte — APPROVED** [review]
works great thanks so much!

**Pull request merged**